.circleci/config.yml

```diff
@@ -113,7 +119,7 @@ workflows:
             branches:
               only: develop
       - deploy:
-          name: deploy_production
+          name: deploy_propuction
```

Comment:
I think `production` is a better name after all ;)

Reply:
I reverted it and things broke so im keeping it for good luck 😂